test/runtests.jl

```diff
@@ -885,6 +885,14 @@ end
             @test length(verbose_err.verbose_msg) > length(err.msg)
 
             close(result)
+
+            result = execute(conn, "SELECT 1;")
```

Comment:
how does this end up triggering an unknownerror?

Reply:
> @test occursin("PGresult is not an error result", unknown_error.verbose_msg)

It doesn't, I'm pulling error information where there is no error. I'm not testing the exact case I want to cover (https://github.com/invenia/LibPQ.jl/issues/116) since I don't know how to reproduce that.

Reply:
ahhh ok I see now. I guess the creator of the #116 issue doesn't quite know how to reproduce.